### DISCUSSION TITLE
run on colab and error

### DISCUSSION THREAD

**Discussion by alinemati45** [csgoh/roadmapper#34] [General]
thanks a lot for your afford.

I ran it on google colab, there is an issue here:

OSError                                   Traceback (most recent call last)
[<ipython-input-9-5336c9239dd2>](https://localhost:8080/#) in <cell line: 2>()
      1 roadmap = Roadmap(400, 400, colour_theme="BLUEMOUNTAIN")
----> 2 roadmap.set_title("My Demo Roadmap")

6 frames
[/usr/local/lib/python3.10/dist-packages/PIL/ImageFont.py](https://localhost:8080/#) in __init__(self, font, size, index, encoding, layout_engine)
    246                         load_from_bytes(f)
    247                     return
--> 248             self.font = core.getfont(
    249                 font, size, index, encoding, layout_engine=layout_engine
    250             )
![image](https://github.com/csgoh/roadmapper/assets/15809162/0031563b-8f4e-414c-bdd5-a9939f41fa69)

**Comment by csgoh** [comment]
Hi, it seems like your notebook is running on Linux platform. Please see this page (https://github.com/csgoh/roadmapper/wiki/installation) for additional steps to install Arial font on Linux. This is a known dependency for now, and the aim is to remove the reliance on the 'Arial' font for the Linux platform in the next release.

**Reply by alinemati45** [reply]
Nope. I simply run it on google colab and on windows 11.
Thanks in advance.

**Reply by alinemati45** [reply]
Please consider below link to check if i am missing something.

https://colab.research.google.com/drive/1-ZJyoL2jllza5ATF3krXTfsfbYE7mvLf?usp=sharing

**Reply by csgoh** [reply]
Referring to your screenshot, the file path shows that you are running your notebook on Linux. So my suggestion to install Arial font on linux still applies. 😉
<img width="986" alt="image" src="https://github.com/csgoh/roadmapper/assets/20867152/6fb0c721-727f-42e6-9f03-4e70fdb5baef">

**Reply by csgoh** [reply]
I managed to get it to work. See the updated notebook: https://colab.research.google.com/drive/1-ZJyoL2jllza5ATF3krXTfsfbYE7mvLf?usp=sharing

Few things to add/change:
* Install Arial font onto Linux OS
* Add "matariki-tech-logo.png" file to your drive
<img width="1020" alt="image" src="https://github.com/csgoh/roadmapper/assets/20867152/61f9780f-9084-4f18-baa3-2646d9abdb09">

**Reply by felix-seifert** [reply]
With the latest release, version `1.2.1`, there should be no need for a Linux-specific setup anymore. If you do not specifically add the Arial dependency, Roadmapper will use an available font from your OS.
